### ISSUE TITLE
Old training data not removed when changing dataset

### ISSUE THREAD

**Issue by rachelglenn** [aecelaya/MIST#4]
When I switch datasets, the preprocessing adds the new dataset to the old dataset directories
numpy/images
numpy/labels

If we can remove these directories to avoid this issue, when running the preprocessing, it would really help. 

https://github.com/aecelaya/MIST/blob/9824ba8e6ff728586c14afe4e26992426a7e737b/mist-torch/runtime/utils.py#L30

def create_empty_dir(path):
    if not (os.path.exists(path)):
        os.makedirs(path)
    else:
      os.system("rm -rf " + path)
      os.makedirs(path)
